### PR TITLE
Remove team_id header

### PR DESCRIPTION
**Pull request by igorosip0v**
This PR: 
- Removes checks of `X-Hasura-Team-Id` on Hasura side
- Remove using `team_id` header from web app
- Refactors some queries and client code to get rid of `team_id` header
- Updates schema

### CODE REVIEW

File: hasura/metadata/databases/default/tables/public_user.yaml [hunk 1]
```diff
@@ -56,11 +56,10 @@ select_permissions:
           - id:
               _eq: X-Hasura-User-Id
           - memberships:
-              _and:
-                - user_id:
+              team:
```
Comment (andy-t-wang) [inline hasura/metadata/databases/default/tables/public_user.yaml:59]: can we not go straight from team here? It is a foreign key in the table

File: web/api/helpers/jwts.ts [hunk 2]
```diff
@@ -98,10 +97,9 @@ export const getUserJWTPayload = (user_id: string, team_id: string) => ({
  */
 export const generateUserJWT = async (
   user_id: string,
-  team_id: string,
-  expiration: number = dayjs().add(7, "day").unix(),
+  expiration: number = dayjs().add(1, "minute").unix(),
```
Comment (andy-t-wang) [inline web/api/helpers/jwts.ts:100]: why are we expiring this after 1 minute?

File: web/tests/integration/user.test.ts [hunk 3]
```diff
@@ -253,10 +253,10 @@ describe("user role", () => {
 
     const response = await client.query({
       query,
+
       variables: {
         id: userFromTeam1?.user_id,
       },
-      context: { headers: { team_id: teams[1].id } }, // Pass a malicious header
```
Comment (andy-t-wang) [inline web/tests/integration/user.test.ts:259]: this test will pass now because we are not spoofing anything. This test doesn't cover anything anymore

Comment (igorosip0v) [reply web/tests/integration/user.test.ts:259]: Removed this one

File: hasura/metadata/databases/default/tables/public_user.yaml (hunk 1 above)
Comment (andy-t-wang) [reply hasura/metadata/databases/default/tables/public_user.yaml:59]: Ah looks like it is null, we should drop this relationship in user then to team

**Review by andy-t-wang — APPROVED** [review]

**Pull request merged**